src/main.rs

```diff
@@ -7,39 +7,39 @@ use std::net::IpAddr;
 use std::path::PathBuf;
 
 use log::*;
-use structopt::*;
 
 use gatekeeper as gk;
 
-#[derive(StructOpt, Debug)]
-#[structopt(name = "gatekeeper")]
+#[derive(clap::Parser, Debug)]
```

Comment:
`-V` option seems to be deleted.

```
gatekeeper$ cargo run -- -V
   Compiling gatekeeper v2.2.0 (/home/eldesh/work/gatekeeper)
    Finished dev [unoptimized + debuginfo] target(s) in 3.27s
     Running `target/debug/gatekeeperd -V`
gatekeeperd
error: unexpected argument '-V' found

Usage: gatekeeperd [OPTIONS]

For more information, try '--help'.
```

Reply:
https://github.com/Idein/gatekeeper/pull/61/commits/21d64f3c42b5f8349ef313408230e4e350551193
fixed